src/sentry/static/sentry/app/views/organizationEvents/index.jsx

```diff
@@ -71,7 +68,25 @@ class OrganizationEventsContainer extends React.Component {
       updateParams: this.updateParams,
     };
 
-    this.state = OrganizationEventsContainer.getInitialStateFromRouter(props);
+    const values = OrganizationEventsContainer.getStateFromRouter(props);
+    this.state = {
+      ...values,
+      queryValues: {
+        ...values,
+      },
+    };
+  }
+
+  componentWillReceiveProps(nextProps, nextState) {
+    if (this.props.location !== nextProps.location) {
+      const values = OrganizationEventsContainer.getStateFromRouter(nextProps);
```

Comment:
should be `this.getStateFromRouter()`

Reply:
It's a static method so you have to use class name